tests/trainer/logging_tests/test_train_loop_logging_1_0.py

```diff
@@ -682,3 +682,69 @@ def get_expected_output(func_attr, original_values):
             assert func_name in trainer.logger_connector.progress_bar_metrics
         else:
             assert func_name not in trainer.logger_connector.progress_bar_metrics
+
+
+def test_logging_sync_dist_true_cpu(tmpdir):
```

Comment:
can we parametrize this test and test both True/False
also, it seems as the bellow is using the very same class, can we define it just once?
cc: @SeanNaren